en/micro-integrator/docs/references/mediators/enrich-Mediator.md

```diff
@@ -426,6 +450,48 @@ In this example, we will replace the key name `name` of every student with `firs
 }
 ```
 
+### Example 13 - Enriching JSON primitive values
+
+We can use property mediators with the `JSON` data type to enrich any JSON primitive, object, or an array to a given target.
+
+!!! Note 
+    When we use a property with the `STRING` data type in the Enrich mediator, it will support native JSON capabilities 
```

Comment:
Please check if didn't change the meaning here...

Reply:
When the property mediator data type is STRING it will support native JSON capabilities only if the string inside the property is JSON object or JSON array. I think we need to highlight that restriction.

Reply:
When the property mediator data type is STRING it will support native JSON capabilities only if the string inside the property is JSON object or JSON array. I think we need to highlight that restriction.